ADVISORS.md

```diff
@@ -1,7 +1,8 @@
 # Members of Advisory Committee
-
-|  Name  |  Company  | Github ID  or Email   |
-|---|---|---|
-| Sandro Mazziotta | Red Hat | smazziot@redhat.com |
-| Eun K Lee | IBM | eunkyung.lee@us.ibm.com |
-| Cathy Zhang |  Intel | cathy.h.zhang@intel.com |
+<!-- markdownlint-disable  MD034 -->
```

Comment:
Which lint are we disabling and why?

Reply:
These indicators are temporarily disabling markdown rule(s). in this case it's the bare urls for the email addresses.
In others it's the line length for certain tables so that they remain human readable. The other rule that we are always hitting in a lot in a lot of the documentation is multiple top level headers (in some cases it makes sense to keep them). 

I could try to add comments in the following format:

```
[This is a comment that will be hidden.]: # 
```

WDYT?